src/sagemaker_sklearn_extension/preprocessing/encoders.py

```diff
@@ -423,3 +423,158 @@ def inverse_transform(self, y):
 
     def _more_tags(self):
         return {"X_types": ["1dlabels"]}
+
+
+class RobustOrdinalEncoder(OrdinalEncoder):
+    """Encode categorical features as an integer array.
+
+    The input to this transformer should be an array-like of integers or
+    strings, denoting the values taken on by categorical (discrete) features.
+    The features are converted to ordinal integers. This results in
+    a single column of integers (0 to n_categories - 1) per feature.
+
+    Read more in the :ref:`User Guide <preprocessing_categorical_features>`.
```

Comment:
This description is a little inaccurate (and is a copy and paste from [sklearn OrdinalEncoder docstring](https://github.com/scikit-learn/scikit-learn/blob/bf24c7e3d/sklearn/preprocessing/_encoders.py#L547-L554)). 

Please reference OrdinalEncoder's docstring and update the description. I recommend taking a look at [RobustLabelEncoders docstring](https://github.com/aws/sagemaker-scikit-learn-extension/blob/e7a7678befa48369eb25a916856ed973c965c311/src/sagemaker_sklearn_extension/preprocessing/encoders.py#L161-L169)

Reply:
committed new docstring